### ISSUE TITLE
Check and fix CORS for REST API

### ISSUE THREAD

**Issue by gorbunkov** [jmix-framework/jmix#1941]
Jmix: 2.0.999-SNAPSHOT

It seems that CORS settings described in the documentation doesn't work. We need to check it and fix.

**Comment by BryanYin** [comment]
Hi, @gorbunkov 

Do we have ETA for this issue or workaround? 

Some customers already started to use REST in Jmix 2 and reported `oauth2/token` endpoint has cors issue.

**Comment by gorbunkov** [comment]
Hi @BryanYin,

I hope we'll deliver the fix in Jmix 2.0.2 in a couple of weeks.

Until then, the following WA should work: create two configurers that will extend the Authorization Server security configuration:

```java
@Component
@Qualifier(AuthServerAutoConfiguration.AuthorizationServerSecurityConfiguration.SECURITY_CONFIGURER_QUALIFIER)
public class AuthServerSecurityConfigurer extends AbstractHttpConfigurer<AuthServerSecurityConfigurer, HttpSecurity> {
    @Override
    public void init(HttpSecurity http) throws Exception {
        http.cors(Customizer.withDefaults());
    }
}
```

```java
@Component
@Qualifier(AuthServerAutoConfiguration.ResourceServerSecurityConfiguration.SECURITY_CONFIGURER_QUALIFIER)
public class ResourceServerSecurityConfigurer extends AbstractHttpConfigurer<ResourceServerSecurityConfigurer, HttpSecurity> {
    @Override
    public void init(HttpSecurity http) throws Exception {
        http.cors(Customizer.withDefaults());
    }
}
```

**Comment by gorbunkov** [comment]
CORS issue should be fixed. By default all request to token endpoints are allowed.
To configure CORS settings, application properties with keys starting with `jmix.cors.*` must be used, e.g.:
```
jmix.cors.allowed-origins = https://www.allowed1.com, https://www.allowed2.com
```
